src/main/java/com/extrahardmode/features/AnimalCrowdControl.java

```diff
@@ -74,14 +74,20 @@ private int getCurrentDensity(Entity e) {
     @EventHandler
     public void onAnimalOverCrowd(CreatureSpawnEvent event) {
         final Entity e = event.getEntity();
-        World world = e.getWorld();
+        final World world = e.getWorld();
 
         final boolean animalOverCrowdControl = CFG.getBoolean(RootNode.ANIMAL_OVERCROWD_CONTROL, world.getName());
         final int threshold = CFG.getInt(RootNode.ANIMAL_OVERCROWD_THRESHOLD, world.getName());
 
         //First check if config allow this feature
         if (!animalOverCrowdControl) return;
         //Get nearby entities from newly spawn animals
+        
+        //Just to check if animal is part of a Pet Plugin assuming spawned pet have nametags already given
+        if(e.getCustomName()!= null) {
+            Bukkit.broadcastMessage(e.getCustomName());
```

Comment:
I'm assuming you're going to change this/using this to test, yes? Doesn't match up to your commit description.